README.md

```diff
@@ -195,32 +196,42 @@ $ sw-precache --root=dist --static-file-globs='dist/**/*.html'
 to your shell (such as the `*` characters in the sample command line above,
 for example).
 
-Finally, there's support for storing a complex configuration in an external
-JSON file, using `--config <file>`. Any of the options from the file can be
-overridden via a command-line flag. For example,
+Finally, there's support for storing a complex configuration as the
+[`module.exports`](https://nodejs.org/api/modules.html#modules_module_exports)
+of an external JavaScript file, using `--config <file>`. Any of the options from
+the file can be overridden via a command-line flag. For example,
 
```

Comment:
_Some suggested text edits._

Finally, there's support for passing complex configurations using `--config <file>`. Any of the options from the file can be overridden via a command-line flag. We strongly recommend passing it an external JavaScript file defining config via module.exports. For example: ...

This provides a great deal of flexibility, such as providing a regular expression for the `runtimeCaching.urlPattern` option. 

We also support passing in JSON for config, however this provides more limited flexibility. [Show an example of this too].